templates/entrypoints/tmux.bash

```diff
@@ -13,4 +15,5 @@ tmux set-option -g status-right ""
 # Set unique to container background color.
 tmux set -g status-bg blue
 # Attach.
-exec tmux attach -t $CONTAINER_NAME
+#exec tmux attach -t $CONTAINER_NAME
+exec sleep +Inf
```

Comment:
Need a better approach here.

I think using something like dumb-init or catononit may make me feel better about this.